atomicapp/nulecule/lib.py

```diff
@@ -0,0 +1,64 @@
+from atomicapp.constants import GLOBAL_CONF, DEFAULT_PROVIDER
+from atomicapp.utils import Utils
+from atomicapp.plugin import Plugin
+
+plugin = Plugin()
+plugin.load_plugins()
+
+
+class NuleculeBase(object):
+    def load(self):
+        pass
+
+    def load_config(self, config={}):
+        config = config or {}
```

Comment:
My guess it, that we would get into trouble if user provides `None` as value and this avoids it..